ios-base/Networking/Extensions/APIClient+Application.swift

```diff
@@ -0,0 +1,10 @@
+import RSSwiftNetworking
+
+/// Provides an easy to access APIClient implementation to use across the application
```

Comment:
```suggestion
/// Provides an easy access to APIClient implementation to use across the application
```

or

```suggestion
/// Provides an easy way to access APIClient implementation to use across the application
```

Reply:
this is not what the sentence means. _easy to access_ means that the APIClient implementation is the one being that way. However the `to` might be unnecessary here, will change that.